docs/tutorial/t-create-client-credentials.md

```diff
@@ -0,0 +1,399 @@
+
+Use "keytool -?, -h, or --help" for this help message
+Use "keytool -command_name --help" for usage of command_name.
+Use the -conf <url> option to specify a pre-configured options file.
+```
+
+### Kafka CLI Client
+
+To verify the credentials we will generate are properly working we will be installing a Kafka CLI client.
+
+```bash
+$ sudo snap install --edge charmed-kafka
+
+# Verify the snap has been installed
+$ snap info charmed-kafka
+name:      charmed-kafka
+summary:   Charmed Kafka for Charmed Operators
+publisher: Canonical Data Platform (dataplatformbot)
+store-url: https://snapcraft.io/charmed-kafka
+contact:   https://chat.charmhub.io/charmhub/channels/data-platform
+license:   unset
+description: |
+  This is a snap that bundles Kafka together with other tools of its
+  ecosystem in order to be used in Charmed Operators, providing automated
+  operations management from day 0 to day 2 on the Apache Kafka event
+  streaming platform deployed on top of a Virtual Machine cluster and K8s
+  cluster. It is an open source, end-to-end, production ready data platform
+  on top of cloud native technologies.
+
+  This bundle together with its charm comes with features such as:
+
+  * Fault-tolerance, replication, scalability and high-availability
+  out-of-the-box.
+  * SASL/SCRAM auth for Broker-Broker and Client-Broker authenticaion enabled
+  by default.
+  * Access control management supported with user-provided ACL lists.
+commands:
+  - charmed-kafka.acls
+  - charmed-kafka.cluster
+  - charmed-kafka.configs
+  - charmed-kafka.console-consumer
+  - charmed-kafka.console-producer
+  - charmed-kafka.consumer-groups
+  - charmed-kafka.consumer-perf-test
+  - charmed-kafka.dump-log
+  - charmed-kafka.get-offsets
+  - charmed-kafka.kafka-streams-application-reset
+  - charmed-kafka.leader-election
+  - charmed-kafka.log-dirs
+  - charmed-kafka.producer-perf-test
+  - charmed-kafka.reassign-partitions
+  - charmed-kafka.replica-verification
+  - charmed-kafka.run-class
+  - charmed-kafka.storage
+  - charmed-kafka.topics
+  - charmed-kafka.transactions
+  - charmed-kafka.trogdor
+  - charmed-kafka.verifiable-consumer
+  - charmed-kafka.verifiable-producer
+  - charmed-kafka.zookeeper-shell
+services:
+  charmed-kafka.daemon: simple, disabled, inactive
+snap-id:      PEGF4Of4BpcyjkBco8QMTwQo0kkrv8c3
+tracking:     3/edge
+refresh-date: 9 days ago, at 13:47 -05
+channels:
+  3/stable:    –
+  3/candidate: –
+  3/beta:      –
+  3/edge:      3.3.2 2023-03-28 (16) 167MB -
+installed:     3.3.2            (16) 167MB -
+```
+
+> At the moment of writing this tutorial the charmed-kafka snap is not yet available on the stable channel. Thus, we will be using the edge channel.
+
+Snaps have constrained access to files as they are designed focused on security. So, once we have created a set of client credentials we will be copying those files into a folder path that the snap has access to read and write, else you might get a permission denied error.
```

Comment:
```suggestion
Snaps have constrained access to files as they are designed with a focus on security. Once we have created a set of client credentials, we will be copying those files into a folder path that the snap has access to read and write to avoid permission denied errors.
```